{"uuid": "b52896b4-d4b7-4be5-b1fd-00cccd748a88", "children": ["f9d99892-3d08-4f2a-8a96-f2401a693339", "4e49ac91-ee06-4399-bbc9-bb2e7fb7d7fd", "2333e01b-e4b2-484a-b29e-4de1912a717f", "9f4d87a2-8f82-4500-99bd-f6b1511ab275", "27d012b6-e7bc-47fc-8492-4fc4da53f6fd"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/tests/TestJobsValidator/test_api.py\", line 30, in setup\n    peviitor_jobs_data = TestScrapers.setup_tests.get_jobs_careers(scraper_class)\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/tests/TestJobsValidator/test_api.py\", line 17, in get_jobs_careers\n    self.scraper_data = self.jobdetails.scrape_peviitor(scraper_class)\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/tests/TestJobsValidator/api_utils/job_details_utils.py\", line 91, in scrape_peviitor\n    response_data = JobDetails._get_request(company_name)\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/tests/TestJobsValidator/api_utils/job_details_utils.py\", line 47, in _get_request\n    updateapi.get_token()\n  File \"/home/runner/work/RaresTestHub/RaresTestHub/tests/TestJobsValidator/api_utils/peviitor_update.py\", line 38, in get_token\n    self.access_token = requests.request(\"POST\", \"https://api.laurentiumarian.ro/get_token\", headers=post_header, data=payload).json()['access']\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/requests/models.py\", line 975, in json\n    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)\n"}, "start": 1734153683336, "stop": 1734153683559}], "start": 1734153683336, "stop": 1734153683723}